doc/api/globals.md

```diff
@@ -45,14 +51,17 @@ References to `__dirname` within `b.js` will return
 will return `/Users/mjr/app`.
 
 ## \_\_filename
+<!-- YAML
+added: v0.0.1
+-->
 
 <!-- type=var -->
 
 * {String}
 
-The filename of the code being executed.  This is the resolved absolute path
-of this code file.  For a main program this is not necessarily the same
-filename used in the command line.  The value inside a module is the path
+The filename of the code being executed. This is the resolved absolute path
+of this code file. For a main program this is not necessarily the same
+filename used in the command line. The value inside a module is the path
```

Comment:
No strong opinion on this so yeah I can revert this change but when rendered it uses a single space anyway so I don't see the point of using 2+.